backend/src/migration/1581033024979-init.ts

```diff
@@ -12,10 +11,14 @@ export class init1581033024979 implements MigrationInterface {
       `CREATE TABLE "high_scores" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "botName" character varying(300) NOT NULL, "score" integer NOT NULL, "createTimeStamp" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP, "updateTimeStamp" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP, CONSTRAINT "PK_da1bb900eb93df2f2d5103d8545" PRIMARY KEY ("id"))`,
       undefined,
     );
+    await queryRunner.query(
```

Comment:
We have to create a new migration for this table. This migration has already been run in all environments.

Reply:
Added new migrations.